overlay/acm/acm-2.14/releaseplan-dev.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+metadata:
+ labels:
+   release.appstudio.openshift.io/auto-release: 'true'
+   release.appstudio.openshift.io/standing-attribution: 'true'
+   release.appstudio.openshift.io/releasePlanAdmission: 'rpa-dev-acm-214'
+ name: dev-publish-acm-214
```

Kustomize would append `-acm-214` making the title `dev-publish-acm-214-acm-214`
```suggestion
 name: dev-publish
```